### PR TITLE
fix: update dated peer deps in eslint config

### PR DESCRIPTION
**Pull request by SimenB**
I wanna bump these in Jest.

(note that a new major of `babel-eslint` is coming any day now as well)

### CODE REVIEW

File: packages/eslint-config-fbjs/package.json [hunk 1]
```diff
@@ -19,12 +19,12 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "peerDependencies": {
-    "babel-eslint": "^7.2.3 || ^8.0.0 || ^9.0.0",
+    "babel-eslint": "^7.2.3 || ^8.0.0 || ^9.0.0 || ^10.0.0",
```
Comment (SimenB) [inline packages/eslint-config-fbjs/package.json]: 11 is in beta, but it requires a babel config file. not sure if you wanna handle that somehow?

Comment (arcanis) [reply packages/eslint-config-fbjs/package.json]: Do you need the 11 for Jest, or would the 10 be enough for now?

Comment (SimenB) [reply packages/eslint-config-fbjs/package.json]: It's needed for TypeScript, but we'll probably go for https://github.com/typescript-eslint/typescript-eslint/ and not `babel-eslint`

**Comment by SimenB** [comment]
Fixed by #343

**Pull request closed without merging**